index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500&family=Pacifico&family=Rubik+Moonrocks&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+    <title>Portfolio Project</title>
+</head>
+
+<body>
+    <div class="top-bar">
+        <h4>Welcome</h4>
+        <img class="menu" src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAARMAAAC3CAMAAAAGjUrGAAAAMFBMVEX///8AAADl5eX09PTV1dUICAiurq5TU1Orq6tJSUnc3Nzt7e1ERET6+vpRUVG9vb0gCS9GAAAA8ElEQVR4nO3dOQ4CQRAEQZblPv//W2yUFsLoNSKekGb1SLPbAQAAAAAAAAAAAADfbodh9+kC9ViGXaYL1Hm6yWm6QGlSmpQmpUlpUpqUJqVJaVKalCalSWlSmpQmpUlpUpqUJnWdbrLBjXqd9pwuAAAAAMCvzstx1Ba3x9f09rjBJjbq0qQ0KU1Kk9KkNClNSpPSpDQpTUqT0qQ0KU1Kk9KkNClvQet9mnW5TRcAAAAAgP/d98PW6QJljy27fWlSmpQmpUlpUpqUJqVJaVKalCalSWlSmpQmpUlpUpqUJuWPs3LLAAAAAAAAAAAAAGAbPhzZL1DD5UvKAAAAAElFTkSuQmCC"
+            alt="Menu Icon">
+    </div>
+    <main>
+        <div class="space"></div>
+        <nav>
+            <header>
+                Hey there. I'm Cisco.
+            </header>
+        </nav>
+        <nav>
+            <article>
+                I'm a software developer.
+            </article>
+        </nav>
+        <div>
+            <p>I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need developed, don't hesitate to contact me.
+            </p>
+        </div>
+        <div class="images">
+            <a href="#"><img src="Images/twittertwitter.png" alt="Twitter logo"></a>
+            <a href="#"><img src="Images/Linkedin.png" alt="LinkedIn logo"></a>
+            <a href="#"><img src="Images/mediummedium.png" alt=" Medium logo"></a>
+            <a href="#"><img src="Images/githubgithub.png" alt="Github logo"></a>
+            <a href="#"><img src="Images/angellistangelist.png" alt="Angellist logo"></a>
+        </div>
```

Comment:
Kindly make this a list using a `ul` tag and wrap each individual icon in an `li` tag for semantic reasons 😃